src/main/javascript/crypto/package.json

```diff
@@ -52,11 +53,14 @@
     "crypto": false
   },
   "dependencies": {
+    "@ethereum-attestation-service/eas-sdk": "^0.28.3",
     "@peculiar/asn1-schema": "^2.3.3",
     "elliptic": "^6.5.3",
     "ethers": "^5.7.2",
+    "js-base64": "^3.7.5",
```

Comment:
we can use import {base64ToUint8array, uint8arrayToBase64} from "../libs/utils"
and avoid to use additional module @micwallace